src/pages/track-page/components/mobile/TrackHeader.tsx

```diff
@@ -11,6 +11,7 @@ import { SquareSizes, CoverArtSizes } from 'common/models/ImageSizes'
 import { FieldVisibility, Remix } from 'common/models/Track'
 import { OverflowAction } from 'common/store/ui/mobile-overflow-menu/types'
 import { squashNewLines } from 'common/utils/formatUtil'
+import { getCannonicalName } from 'common/utils/genres'
```

Comment:
Probably not in this PR but this name feels like a typo, I think it should be `getCanonicalName`?

Reply:
Haha good catch, let me change that rq